### PR TITLE
search judicial

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5472

### Change description ###
Refactoring find person(for hearing only judicial users) component

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/prd/judicial/index.ts [hunk 1]
```diff
@@ -19,7 +19,23 @@ export async function searchJudicialUserByPersonalCodes(req: EnhancedRequest, re
...
+  }
+}
+
+/**
+ * @overview getJudicialUsersSearch from searchString, i.e. jam
+ * @description API sample: POST /api/prd/judicial/users/search
+ * @example with body {searchString: jam}
+ */
+export async function getJudicialUsersSearch(req: EnhancedRequest, res: Response, next: NextFunction) {
+  const reqBody = req.body;
```
Comment (rayliangatos) [inline api/prd/judicial/index.ts:35]: does the body havethe serviceCode

File: src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.html [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
-  <div class="person-button">
-    <div class="govuk-button govuk-button--secondary" data-module="govuk-button" (click)="excludeJudge();">{{submitButtonName}}</div>
+  <span id="validation-error" class="govuk-error-message" *ngIf="personFormGroup.controls.selectedFormControl.errors">
+    <span class="govuk-visually-hidden">Error:</span>{{personFormGroup.controls.selectedFormControl.errors.error }}
+  </span>
+  <div class="exclude-person" [class.column-view]="isColumnView">
+    <div class="person-input">
+      <xuilib-search-judicials id="judicialsSearch" [placeholderContent]="'e.g. John Smith'" [idValue]="idValue"
+        [selectedJudicials]="judgeList" [control]="personFormGroup.controls.selectedFormControl">
+      </xuilib-search-judicials>
```
Comment (rayliangatos) [inline src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.html:13]: if here can pass the serviceCode

**Review by rayliangatos — APPROVED** [review]
Approved will address the serviceCode in integration

File: api/prd/judicial/index.ts (hunk 1 above)
Comment (rayliangatos) [reply api/prd/judicial/index.ts:35]: will resolve in integration

File: src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.html (hunk 2 above)
Comment (rayliangatos) [reply src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.html:13]: will resolve in integration

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Pull request merged**